daco2ego/templates/secret.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  password: {{ .Values.configFileContent | b64enc | quote }}
+  json-config: {{ .Values.configFileContent | b64enc | quote }}
```

Comment:
I think we need to name this `default.conf` and udpate the mountPath in `templates/cronjob.yaml` to be `/daco2ego/config`

Reply:
Addressed.